<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打字训练器</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;方正小标宋简体&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>打字训练器</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_left">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_right">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一句</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string>此次训练的句子为：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_left">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 11pt &quot;方正小标宋简体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
